tools/mull-runner/mull-runner.cpp

```diff
@@ -259,7 +275,7 @@ int main(int argc, char **argv) {
     stringstream << "Surviving mutants: " << surviving;
     diagnostics.info(stringstream.str());
 
-    if (!tool::AllowSurvivingEnabled) {
+    if ((!tool::AllowSurvivingEnabled) && (tool::MutationScoreThreshold > score)) {
```

Comment:
Could you please add/extend a test case to check when "allow surviving" is enabled and the score is exactly 100? I guess this `>` should be `>=`.

Reply:
You're right, apologies for the confusion, it was a long day 😃